sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -759,6 +759,14 @@ object RapidsConf {
     .stringConf
     .createOptional
 
+  val CUDF_VERSION_OVERRIDE = conf("spark.rapids.cudf-version-override")
```

Comment:
Everything except for shims-provider-override is camelCase just like with spark. Can we update this to be the same?